Packs/Zimperium/Integrations/ZimperiumV2/ZimperiumV2_test.py

```diff
@@ -0,0 +1,353 @@
+
+
+def test_policy_app_settings_get_command(client, requests_mock):
+    """
+        When: running zimperium-policy-app-settings-get command.
+        Given: policy_id to get information about.
+        Then: validate the command result returned.
+        """
+    args = {'app_settings_policy_id': '9e'}
+    mock_response_policy_app_settings = util_load_json(
+        './test_data/policy_app_settings.json')
+
+    requests_mock.get(f'{SERVER_URL}/mtd-policy/public/v1/app-settings/policies/9e', json=mock_response_policy_app_settings)
+    results = policy_app_settings_get_command(client=client, args=args)
+
+    assert results.outputs_prefix == 'Zimperium.PolicyAppSetting'
+    assert results.outputs.get('id') == mock_response_policy_app_settings.get('id')
+    assert results.outputs_key_field == 'id'
+    assert 'Policy App Settings' in results.readable_output
+    assert results.raw_response == mock_response_policy_app_settings
+
+
+def test_policy_device_inactivity_list_command(client, requests_mock):
+    """
+        When: running zimperium-policy-device-inactivity-list command.
+        Given: team_id to filter by.
+        Then: validate the command result returned.
+        """
+    args = {'team_id': '33'}
+    mock_response_policy_device_inactivity_list = util_load_json(
+        './test_data/policy_device_inactivity_list.json')
+
+    requests_mock.get(f'{SERVER_URL}/devices/public/v1/dormancy/policies', json=mock_response_policy_device_inactivity_list)
+    results = policy_device_inactivity_list_command(client=client, args=args)
+
+    assert results.outputs_prefix == 'Zimperium.PolicyDeviceInactivity'
+    assert results.outputs[0].get('id') == mock_response_policy_device_inactivity_list[0].get('id')
+    assert results.outputs_key_field == 'id'
+    assert 'Device Inactivity' in results.readable_output
+    assert results.raw_response == mock_response_policy_device_inactivity_list
+
+
+def test_policy_device_inactivity_get_command(client, requests_mock):
+    """
+        When: running zimperium-policy-device-inactivity-get command.
+        Given: policy_id to get information about.
+        Then: validate the command result returned.
+        """
+    args = {'policy_id': 'ff'}
+    mock_response_policy_device_inactivity_get = util_load_json(
+        './test_data/policy_device_inactivity_get.json')
+
+    requests_mock.get(f'{SERVER_URL}/devices/public/v1/dormancy/policies/ff', json=mock_response_policy_device_inactivity_get)
+    results = policy_device_inactivity_get_command(client=client, args=args)
+
+    assert results.outputs_prefix == 'Zimperium.PolicyDeviceInactivity'
+    assert results.outputs.get('id') == mock_response_policy_device_inactivity_get.get('id')
+    assert results.outputs_key_field == 'id'
+    assert 'Device Inactivity' in results.readable_output
+    assert results.raw_response == mock_response_policy_device_inactivity_get
+
+
+@freeze_time("2023-12-12 15:00:00 GTM")
+@pytest.mark.parametrize(
```

Comment:
please split to different methods as recently announced in the retro (DM if you are unaware and want more details)